src/context/reducer.ts

```diff
@@ -1,10 +1,4 @@
-import { StateType, TransactionType } from "./state";
-
-export type ActionType = {
-  type: "setTransactions";
-  transactions: TransactionType[];
-  transactionsFetched: StateType["transactionsFetched"];
-};
+import {ActionType, StateType, TransactionType} from "./types";
 
 export function reducer(state: StateType, action: ActionType): StateType {
```

Comment:
reducers are not context. We should keep these 2 separated.
And if possible, we should use reducers for data from outside, and context for components state... if possible, this is a very intense battle all the time. REdux vS Context... Well. if we use them both we should separate them as they are 2 different things.